featuregate/flag.go

```diff
@@ -49,7 +54,28 @@ func (f *flagValue) Set(s string) error {
 		case '+':
 			id = id[1:]
 		}
-		errs = multierr.Append(errs, f.reg.Set(id, val))
+		gatesEnabled[id] = val
+		if _, ok := f.reg.gates.Load(id); !ok {
+			errs = multierr.Append(errs, fmt.Errorf("no such feature gate %q", id))
+		}
 	}
+	f.reg.VisitAll(func(gate *Gate) {
+		enabled, ok := gatesEnabled[gate.id]
+		if !ok {
+			if *f.strict && (gate.stage == StageAlpha || gate.stage == StageBeta) {
+				errs = multierr.Append(errs, fmt.Errorf("gate %q is in %s and is not explicitly configured", gate.id, gate.stage))
```

Comment:
To make the wording match that of the other errors, we can also phrase this as a 'must':
```suggestion
				errs = multierr.Append(errs, fmt.Errorf("gate %q is in %s and must be explicitly configured", gate.id, gate.stage))
```